policies/feature-branch-approval-policy.md

```diff
@@ -0,0 +1,52 @@
+# Feature Branch Approval Policy
+
+The [Time-based Release Policy] defines a regular twice a year schedule for new
+OpenSSL versions. In the case of large features that cannot reasonably be
+implemented and reviewed within a single PR there is a risk that the feature
+will only be partially implemented and merged to master by the time that a
+feature freeze occurs for the next release.
+
+To avoid this problem feature branches can be used. A feature branch is simply
+a branch in the main git repository that can be used as a target branch for
+PRs related to a single new feature.
+
+A feature branch will be created based on the current state of master at the
+time of its creation. If a feature branch exists for a protracted period of time
+then it will need to be rebased on a regular basis. The process for doing so is
+deferred to other policies.
+
+The creation of a new feature branch implies some overhead for the project to
+manage that feature branch (such as for example periodic rebases, and to review
+PRs targetting it). Such a feature branch would typically only be created where
+it has been agreed that the feature is desirable; exists on the [Project Board];
+is of sufficiently high priority; and there are expected to be resources
+available to work on the feature development and to review it. For this reason
+the creation of new feature branches is controlled via an approval process.
+
+Submitting a request for a new feature branch should be done by creating an
+issue in the [openssl/project] repository. Such a request can be created by
+anyone without restriction. The request should specify:
+
+- The name of the feature branch requested
```

Comment:
My suggestion:
```suggestion
- The name of the feature branch requested in the form of `feature/<name>`
```

This way we will use the same pattern as for the already existing `feature/quic-server` branch.